CHANGELOG.md

```diff
@@ -7,228 +7,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- EIP-1559 - Prevent uncaught exception when passing fees to getGasFeeTimeEstimate ([#11759](https://github.com/MetaMask/metamask-extension/pull/11759))
-- EIP-1559 - Temporarily remove the loading heartbeat from the transaction detail ([#11762](https://github.com/MetaMask/metamask-extension/pull/11762))
-- EIP-1559 - Remove unncessary props from AdvancedFormControls ([#11757](https://github.com/MetaMask/metamask-extension/pull/11757))
-- Ensure gas fees update in popover on poll for new values ([#11760](https://github.com/MetaMask/metamask-extension/pull/11760))
-- remove unnecessary conversion call ([#11742](https://github.com/MetaMask/metamask-extension/pull/11742))
-- Fixing errors in EditGasDisplay ([#11748](https://github.com/MetaMask/metamask-extension/pull/11748))
-- Allow max priority fee to be below 1, and only require it to be greater than 0 ([#11749](https://github.com/MetaMask/metamask-extension/pull/11749))
-- Allows users to set a max priority fee below suggested, just showing a warning in that case ([#11750](https://github.com/MetaMask/metamask-extension/pull/11750))
-- Rename effective gas price field to total gas fee ([#11754](https://github.com/MetaMask/metamask-extension/pull/11754))
-- Fix ipfs dependency vulernability ([#11745](https://github.com/MetaMask/metamask-extension/pull/11745))
-- Remove estimate detail above advanced gas controls in non-1559 network ([#11744](https://github.com/MetaMask/metamask-extension/pull/11744))
-- Fall back to aggregator name when icon is unavailable (Swaps Load View) ([#11718](https://github.com/MetaMask/metamask-extension/pull/11718))
-- EIP-1559 - Show minium native currency in banner when on testnets ([#11743](https://github.com/MetaMask/metamask-extension/pull/11743))
-- Fix a position of a tooltip icon ([#11739](https://github.com/MetaMask/metamask-extension/pull/11739))
-- Add gasPrice to be used on non-1559 networks for transaction details on confirm screen ([#11741](https://github.com/MetaMask/metamask-extension/pull/11741))
-- EIP-1559 - Provide better validation for gas price and gas limit ([#11736](https://github.com/MetaMask/metamask-extension/pull/11736))
-- EIP-1559 - Fix education modal typo ([#11740](https://github.com/MetaMask/metamask-extension/pull/11740))
-- Ensure that gas price in popover updates when api provided estimate updates ([#11727](https://github.com/MetaMask/metamask-extension/pull/11727))
-- EIP-1559 - Ensure transaction detail font-size and icon colors are consistent with Figma design ([#11735](https://github.com/MetaMask/metamask-extension/pull/11735))
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#11552](https://github.com/MetaMask/metamask-extension/pull/11552): Integrating new TokenRatesController to expand fiat conversion rates for some non-Mainnet networks
```

Comment:
The work described here was done in #12066 :thinking: #11552 was a prerequisite, but it wasn't a user-facing change.